edit rn 13.1.1 (#1642)
mia-platform/documentation: docs/release-notes/v13.1.1.md

## Microfrontend Composer

### Configurator

#### Improvements

##### Template not required when creating a compose page

When creating a new page of type _compose_, choosing a staring template is no longer required. If none is chosen an empty page will be created.

## Marketplace

### Marketplace Updates

#### AI RAG Chat

Our **AI RAG Chat** Application is now available in **stable** release, now featuring automatic embeddings generation from website pages.
For more information on how to configure your custom RAG Application, head to the [related tutorial](/marketplace/tutorials/create-a-rag-application.md).

#### AI RAG Template

The new version `0.3.1` of the AI RAG Template is **now stably available**! 

#### Flow Manager

The new version `2.6.5` of the Flow Manager is available!

##### New Features

- Added possibility to specify an optional JSON schema validation for incoming events payload
- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error

#### CRUD Service

The new version `7.1.1` of the CRUD Service is available!

> ##### Improvements

Reviewer: ```suggestion
##### Bug Fix

```

> * extend to `/bulk` endpoint the requests preprocessing performed on exposed routes of View with lookup feature enabled

Reviewer: ```suggestion
* `/bulk` endpoints exposed by CRUD Views with lookup feature enabled are now working properly
```

## Microfrontend Composer

### Configurator

#### Improvements

##### Template not required when creating a compose page

When creating a new page of type _compose_, choosing a staring template is no longer required. If none is chosen an empty page will be created.

## Marketplace

### Marketplace Updates

#### AI RAG Chat

Our **AI RAG Chat** Application is now available in **stable** release, now featuring automatic embeddings generation from website pages.
For more information on how to configure your custom RAG Application, head to the [related tutorial](/marketplace/tutorials/create-a-rag-application.md).

#### AI RAG Template

The new version `0.3.1` of the AI RAG Template is **now stably available**! 

#### Flow Manager

The new version `2.6.5` of the Flow Manager is available!

##### New Features

- Added possibility to specify an optional JSON schema validation for incoming events payload
- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error

#### CRUD Service

The new version `7.1.1` of the CRUD Service is available!

##### Bug Fix

> 

Reviewer: ```suggestion
```

* `/bulk` endpoints exposed by CRUD Views with lookup feature enabled are now working properly

## How to update your Console

For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.2-beta.0`.

> 

Reviewer: ```suggestion
```

### Improvements

#### Flow Manager configurator supports Flow Manager version 2.6.5

No-code configurability has been added for the new features introduced with Flow Manager version `2.6.5`. Namely:

- the possibility to add a JSON schema validation for events payload
- a setting to raise errors on events that are not mapped as outgoing transitions for the current state

### Bug Fix

This version addressed the following bugs:

*

## Fast Data

### Fast Data Runtime Management

#### Bug Fix

## Microfrontend Composer

### Configurator

#### Improvements

##### Template not required when creating a compose page

When creating a new page of type _compose_, choosing a staring template is no longer required. If none is chosen an empty page will be created.

## Marketplace

### Marketplace Updates

#### Flow Manager

The new version `2.6.5` of the Flow Manager is available!

##### New Features

- Added possibility to specify an optional JSON schema validation for incoming events payload
- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error

## How to update your Console

> For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `x.x.x`.

Reviewer 2: @mattia-fumo FYI

> For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `x.x.x`.

Author: ```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.2-beta.0`.

```
